components/is-admin/index.ts

```diff
@@ -8,12 +13,15 @@ import { useSelect } from '@wordpress/data';
  * fallback component via the fallback prop.
  *
  * @param {object} props react props
- * @param {*} props.fallback fallback component
- * @param {*} props.children child components
- * @returns {*}
+ * @param {React.ReactNode|null} props.fallback fallback component
+ * @param {React.ReactNode} props.children child components
+ * @returns {null|React.ReactNode}
```

Comment:
When we are adding types I think we can safely remove the JSDoc comments. 

If we want to add additional context to the type we can add a comment above the type value like so:

```ts
interface IsAdminProps {
    /**
     * fallback component
     */
 	fallback: React.ReactNode | null;

    /**
     * child component
     */
 	children: React.ReactNode;
 }
```